packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -93,6 +95,14 @@ export const options: CliCommandOptions<ChainArgs> = {
     group: "chain",
   },
 
+  serveHistoricalState: {
+    description:
+      "Enable regenerating and loading state to serve historical data. Fetching this data is expensive and may affect validator performance.",
```

Comment:
could simplify for now

```suggestion
      "Enable regenerating state to serve historical data. Fetching this data is expensive and may affect validator performance.",
```

Reply:
maybe mention `finalized` state?

Reply:
so what's interesting is we can actually serve finalized state if we pass `finalized` in the query, because it's a checkpoint state I think that we use

so far we used mostly the term "historical" it's bit vague I guess, can add finalized here